Source/gg2/Scripts/Game/game_init.gml

```diff
@@ -367,13 +367,20 @@ global.launchMap = "";
     /* Windows 8 is known to crash GM when more than three (?) sounds play at once
      * We'll store the kernel version (Win8 is 6.2, Win7 is 6.1) and check it there.
      ***/
-    registry_set_root(1); // HKLM
-    global.NTKernelVersion = real(registry_read_string_ext("\SOFTWARE\Microsoft\Windows NT\CurrentVersion\", "CurrentVersion")); // SIC
-    if(!registry_exists_ext("\SOFTWARE\Microsoft\Windows NT\CurrentVersion\", "CurrentMajorVersionNumber"))
-        global.CurrentMajorVersionNumber = -1;
+    global.CurrentMajorVersionNumber = -1;
+    if(!gm8c_runner_registry_access(GM8C_W32_KEY_READ))
+    {
+        global.NTKernelVersion = -1;
+    }
     else
-        global.CurrentMajorVersionNumber = registry_read_real_ext("\SOFTWARE\Microsoft\Windows NT\CurrentVersion\", "CurrentMajorVersionNumber");
-    
+    {
+        registry_set_root(1); // HKLM
+        global.NTKernelVersion = real(registry_read_string_ext("\SOFTWARE\Microsoft\Windows NT\CurrentVersion\", "CurrentVersion")); // SIC

```

Comment:
It seems we only check this to enable the sound crash workaround. Is that even still an issue with GM8.1?

Reply:
It is, because GM 8.1 uses the same version of DirectX as 8.0 does.